workflows/cloud-client/main.py

```diff
@@ -0,0 +1,65 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+
+def execute_workflow(
+    project, location="us-central1", workflow="myFirstWorkflow"
+):
+    """Execute a workflow and print the execution results."""
+    # [START workflows_api_quickstart]
+    import time
+
+    from google.cloud import workflows_v1beta
+    from google.cloud.workflows import executions_v1beta
+    from google.cloud.workflows.executions_v1beta.types import executions
+
+    if not project:
+        raise Exception('GOOGLE_CLOUD_PROJECT env var is required.')
```

Comment:
Few thoughts.

- If you copy paste this code, I think you'll get NameError for `project` variable.

- If you want to check the value of the project (acquired from `GOOGLE_CLOUD_PROJECT` env var) and display the error message about the env var, you should acquire the value from the env within the sample. Imagine the case where you copy paste this code and encounter the error message about `GOOGLE_CLOUD_PROJECT`. This error message is misleading. 

My suggestion is to remove the `project` from the function.

```python
def execute_workflow(
    location="us-central1", workflow="myFirstWorkflow"
):
    # ...
    # ...
    project = os.environ['GOOGLE_CLOUD_PROJECT'] 
    # ...
```

It will give users an error message for free when `GOOGLE_CLOUD_PROJECT` env var is not set.
If you think the error message is too cryptic, you can do nicer things of course.

WDYT?